app/middlewares/checkSchema.js

```diff
@@ -0,0 +1,10 @@
+const { checkSchema, validationResult } = require('express-validator/check');
+
+const error = require('../errors');
+
+const checkValidationResult = (req, res, next) => {
+  const errors = validationResult(req);
+  return errors.isEmpty() ? next() : res.status(400).json({ errors: errors.array(), ...error.badRequest() });
```

Comment:
You don't need set status. Check `errors.js` and add key `BAD_REQUEST` with status code `400`. Also, `json` is not necessary.  If you want send a json, is more easy, use `.send({ key: value})`.